### PR TITLE
Feature proper readme

### PR DESCRIPTION
**Pull request by tntmeijs**
I've reworked the old README.MD a bit. We now have a lot more information in the README.

### CODE REVIEW

**Comment by tntmeijs** [comment]
One this is approved, we can close #204.

**Comment by VZout** [comment]
Maybe make the feature list a list with checkboxes instead so people know which tasks are done or not. i think github markdown has support for that.

**Comment by VZout** [comment]
Also I can host the media on my website tomorrow so we don't have to download those images as well when we clone.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,157 @@
-# Procedural-Ray-Tracing
-Repository for our Procedural Ray Tracing project, made by Y3 students at Breda University of Applied Science (former NHTV)
-More information: [Wiki](https://teamwisp.github.io/)
+# Wisp Rendering Framework
```
[the remark this answers is not in the document]
Comment (VZout) [reply README.md]: You can change this into a link to the blog maybe.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,157 @@
...
+We are a group of students from [Breda University of Applied Sciences]([https://www.buas.nl](https://www.buas.nl/)) working on a rendering framework using cutting-edge real-time ray-tracing technology from NVIDIA.
+
+## Table of contents
+
+- [Showcase](#showcase)
```
Comment (VZout) [inline README.md]: I would put the showcase further down. If I was a developer looking at this page i would be annoyed at how far I have to scroll for the project description, features and the rest of the info.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (VZout) [reply README.md]: Also add the logo infront of it maybe.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,157 @@
...
+- Shadows.
+- Physically Based Rendering.
+- Translucent & Transparency.
+- GPU Accelerated.
+- Realtime rendering.
+- Built for NVIDIA RTX.
+- Implemented into Maya/Blender’s viewport.
+- Dynamic LOD’s for non-selected / background objects.
+
+## Contributing
```
Comment (VZout) [inline README.md]: I would remove most of this header and just link to the TDD and mention the most important pages. It doesn't make sense for such a niece thing to take up so much space imo.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,157 @@
...
+
+### Code architecture
+
+New contributors may find it a bit overwhelming to get started on a project of this size. This is why we have created a short introduction page on the Wiki that explains some of the basics of the framework: [https://teamwisp.github.io/architecture/](https://teamwisp.github.io/architecture/)
+
+## More information
+
+This README is only a summary of the most important things developers should know about the project. It is highly encouraged to have a look at our [Wiki](https://teamwisp.github.io/). It covers all topics mentioned above, and more...
+
+If you made it this far, congratulations! We hope to see you soon in our development server: https://discordapp.com/invite/KthSUvs
```
Comment (VZout) [inline README.md]: Add a discord logo! it now fades away into all the text. Also maybe put the discord link a bit higher.

also I always hate it when people write stuff like this "If you made it this far, congratulations!"

File: README.md [hunk 5]
```diff
@@ -1,3 +1,157 @@
...
+  - [Setting up your workspace](#setting-up-your-workspace)
+  - [Development pipeline](#development-pipeline)
+  - [Coding standards](#coding-standards)
+  - [Documentation standards](#documentation-standards)
+  - [Code architecture](#code-architecture)
+- [More information](#more-information)
+
+## Showcase
+
+| ![](media/sun_temple.png)                     | ![](media/shadows_reflections_in_reflections.png) |
```
Comment (VZout) [inline README.md]: We can also link the trailer again.

**Pull request merged**